mozdownload/scraper.py

```diff
@@ -457,7 +457,7 @@ def is_build_dir(self, folder_name):
     def get_build_info_for_date(self, date, build_index=None):
         """Return the build information for a given date."""
         url = urljoin(self.base_url, self.monthly_build_list_regex)
-        has_time = date and date.time()
+        has_time = date and date.time() and date.strftime('%H-%M-%S') != '00-00-00'
```

Comment:
Due you know for which value of date this fails? Which test is causing that?

Reply:
Well, I see in the stacktrace that there are already the `args` and values which the tests fail with: https://travis-ci.org/mozilla/mozdownload/jobs/503557491

Reply:
As best pick one of those failing tests and run it alone by using the -k option like `tox -- -k test_name`. Also specify -s so that you get all the output. I would like to know what `date.time()` differs in between Python2.7 and Python3.

Reply:
Test `test_build_indices`

Python 3.6: https://gist.github.com/AndreiH/d8cb89dd509a7a4733d58ba706fbdb27
Python 2.7: https://gist.github.com/AndreiH/7b32e6c9a22cfb2212d58ab5d5602769

Reply:
So here is the snippet for your testing: https://gist.github.com/whimboo/63ac15a7029fa2b6be9da33858fb0a91

Note the different behavior between Python2 and Python3. We have to get it fixed, and comparing to `00:00:00` is not a good idea given that at this second we also could have a sub folder.